### PR TITLE
Adds mongo database to docker compose and initializes it

### PR DESCRIPTION
**Pull request by aduartjs**
## Description

Adds mongodb to the project by configuring it in `docker-compose.yml`
Also, initializes a database with test data.

## Related Issue
#12

## Motivation and Context

A need of having a mongodb database for the project with an easy way of initialization

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/codevsoft/workkit/blob/master/CONTRIBUTING.md) doc document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .mongo/init-mongo.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+mongo <<EOF
+use workkit
+db.createUser({
+  user: 'workkit',
+  pwd: 'workkitpwd',
+  roles: [{
+    role: 'readWrite',
+    db: 'workkit'
+  }]
```
Comment (aduartjs) [inline .mongo/init-mongo.sh:12]: As this is an .sh file, we can use env variables to set the db parameters

File: .mongo/init-mongo.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+set -e
+
+mongo <<EOF
+use workkit
+db.createUser({
+  user: 'workkit',
+  pwd: 'workkitpwd',
+  roles: [{
+    role: 'readWrite',
+    db: 'workkit'
```
Comment (aduartjs) [inline .mongo/init-mongo.sh:11]: As this is an .sh file, we can use env variables to set db parameters

**Review by namelivia — APPROVED** [review]
Nice! :tada:

**Review by 73nko — APPROVED** [review]
LGTM! ✅

**Pull request merged**